acceptance-tests/tests/src/test/resources/ibft/ibft.json

```diff
@@ -10,7 +10,8 @@
     "ibft2": {
       "blockperiodseconds": 1,
       "epochlength": 30000,
-      "requesttimeoutseconds": 5
+      "requesttimeoutseconds": 5,
+      "blockrewardeth": 5
```

Comment:
Rather the default was left as 0 as what most people will use and just modify the blockreward for the block reward AT

Reply:
Yeah agreed - but it's REALLY hard to do - given no existing tests prove that the validator's not paid, I figured this was ok?